tests/common/utils.ts

```diff
@@ -20,6 +20,28 @@ export interface ProofData {
     vk?: string;
 }
 
+export const proofTypes = Object.keys(ProofType).map((key) => ProofType[key as keyof typeof ProofType]);
+export const curveTypes = Object.keys(Groth16CurveType).map((key) => Groth16CurveType[key as keyof typeof Groth16CurveType]);
+
+const seedPhrases = [
+    process.env.SEED_PHRASE_1,
+    process.env.SEED_PHRASE_2,
+    process.env.SEED_PHRASE_3,
+    process.env.SEED_PHRASE_4,
+    process.env.SEED_PHRASE_5,
+    process.env.SEED_PHRASE_6,
```

Comment:
I think this is missing `process.env.SEED_PHRASE_7`

Reply:
7 is more of a generic one used in the unit tests whereas 1 - 6 are for the proof types / variants.  Will add this as explanation.